src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StorageTest {
+
+    @Test
+    public void testSaveAndLoad() throws DukeException, IOException {

```

Comment:
I don't think this follows our coding standard for naming tests, it should follow the format: `featureUnderTest_testScenario_expectedBehavior()`